### DISCUSSION TITLE
how about adding Gitalk or any other comment plugin?

### DISCUSSION THREAD

**Discussion by hyperty** [retypeapp/retype#199] [Ideas]
i think it would be a great function.

**Comment by fabriciomurta** [comment]
That's something I sure heard one of us talking about during our meetings. At some point, Retype will inevitable support embedding comments in articles, one way or another. Thanks for the request, we have logged an entry in our internal issue tracker especially for your request!

**Comment by geoffreymcgill** [comment]
I was able to get the [Disqus](https://disqus.com/) comment plugin working on all content pages by adding the chuck of embed code they provide into the `_includes/body.md` file. 

```html
<!-- Add content to _includes/bottom.md to include here -->
<div id="disqus_thread"></div>
<script>
    (function() {
    var d = document, s = d.createElement('script');
    s.src = 'https://retypeapp.disqus.com/embed.js';
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
    })();
</script>
```

Once added, the Disqus comment widget is added to all pages and seems to work well. 

<img width="941" alt="Screen Shot 2023-03-10 at 5 59 43 PM" src="https://user-images.githubusercontent.com/62210/224455632-01602f61-f360-4f22-8c4b-0e1d6851becc.png">

Hope this helps.
